odoo/models.py

```diff
@@ -6280,6 +6280,10 @@ class TransientModel(Model):
     _abstract = False           # not abstract
     _transient = True           # transient
 
+    @api.autovacuum
+    def _run_transient_vacuum(self):
+        self._transient_vacuum(force=True)
```

Comment:
yup, that's basically was was doing `_gc_transient_models`

Reply:
Might have to be renamed if we choose to follow a convention though.

BTW how about moving the code of `_transient_*` methods directly inside `TransientModel`, i.e. here, rather than keeping them in `BaseModel`? We can keep the names, so that callsites that worked will still work (calling them on non-transient models would raise anyway)

Reply:
> BTW -snip-

Yes :) Duty for another PR ;)